torch/_torch_docs.py

```diff
@@ -1481,6 +1481,134 @@ def merge_dicts(*dicts):
     silently incorrect.
 """)
 
+add_docstr(torch.hsplit,
+           r"""
+hsplit(input, indices_or_sections) -> List of Tensors
+
+Splits :attr:`input`, a tensor with one or more dimensions, into multiple tensors
+horizontally (column wise) according to :attr:`indices_or_sections`. Each split
+is a view of :attr:`input`.
+
+If :attr:`input` is one dimensional this is equivalent to calling
```

Comment:
NumPy's documentation for hsplit (https://numpy.org/doc/stable/reference/generated/numpy.hsplit.html?highlight=hsplit#numpy.hsplit) says "hsplit is equivalent to split with axis=1", why do we say something different?

Reply:
I believe implementation of numpy hsplit and Pytorch split is almost identical. For Numpy they just haven't mentioned about 1-dimensional case separately in documentation.